code/modules/hydroponics/grown.dm

```diff
@@ -107,6 +107,11 @@
 // Various gene procs
 /obj/item/reagent_containers/food/snacks/grown/attack_self(mob/user)
 	if(seed && seed.get_gene(/datum/plant_gene/trait/squash))
+		if(seed.get_gene(/datum/plant_gene/trait/teleport))
+			to_chat(user, span_notice("You start squashing the [src]"))
+			if(do_after(user, 2 SECONDS, src))
+				squash(user)
+			return ..()
 		squash(user)
```

Comment:
```suggestion
```

Reply:
This way it would break every other squashable fruit